plugins/lua/buildingplan/planneroverlay.lua

```diff
@@ -490,19 +514,20 @@ function PlannerOverlay:init()
                 },
                 widgets.CycleHotkeyLabel{
                     view_id='choose',
-                    frame={b=0, l=0},
-                    key='CUSTOM_I',
-                    label='Item selection:',
+                    frame={b=0, l=23},
+                    key='CUSTOM_Z',
+                    label='Choose items:',
+                    label_below=true,
                     options={
-                        {label='Use filters', value=0},
+                        {label='with Filters', value=0},
                         {
                             label=function()
                                 local automaterial = itemselection.get_automaterial_selection(uibs.building_type)
-                                return ('Last choice (%s)'):format(automaterial or 'Will ask')
+                                return ('Last used (%s)'):format(automaterial or 'n/a')
```

Comment:
I settled on "Will ask" because I wanted to convey that it was *ok* to choose this option even if you haven't made a prior selection. "n/a" makes me worried that people will think that this option isn't valid.

Reply:
> "n/a" makes me worried that people will think that this option isn't valid.

that is a valid concern, but i'm not sure how i feel about saying "Will ask" 🤔 
like... "use last filter (will ask)" is like... what?! will ask what? my last filter? i don't have a last filter! 

how can i say, it shouldn't read like that at all! 
i mean, the text in the parentheses should be the last filter, not some reassuring text 🤔 the optimal thing would be to not give you the option of choosing the last used item if you didn't choose any item before!

Reply:
i tried looking into ways to hide that option if the last-used item was not set, but i couldn't find a way to do it ;_;
if we _can't_ hide it, what should the text say? (really, it should be hidden though)